.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
```

Comment:
You can probably leave the assignees off since that's something more easily managed via the github issues page.

Reply:
Done